packages/atclient/src/atnotification.c

```diff
@@ -5,6 +5,11 @@
 
 #define TAG "atnotification"
 
+// Json library specific
+#if ATCLIENT_JSON_PROVIDER == ATCLIENT_JSON_PROVIDER_CJSON
+static int atclient_atnotification_from_cjson_node(atclient_atnotification *notification, const cJSON *root);
+#endif
+
```

Comment:
This function is private now, and conditionally linked into the program only when cJSON is available